src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -60,7 +60,7 @@ def _set_params(
             metric_names += get_multi_channel_template_metric_names()
         metrics_kwargs = metrics_kwargs or dict()
         params = dict(
-            metric_names=[str(name) for name in metric_names],
+            metric_names=[str(name) for name in np.unique(metric_names)],
```

Comment:
If this is a list maybe you can use set?

Reply:
e.g:  `{str(name) for name in metric_names}`

Reply:
well it needs to be a list!

Reply:
Yeah, it could also be:
[[str(name) for name in set(metric_names)]

But it probably does not matter for this size anyway : )